jet-fuel/test/integration/urls_statistics_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.